lib/componentRelay.ts

```diff
@@ -256,16 +256,16 @@ export default class ComponentRelay {
         const originalMessage = this.sentMessages?.filter((message: MessagePayload) => {
           return message.messageId === payload.original?.messageId
         })[0]
-  
-        if (!originalMessage) {
-          // Connection must have been reset. We should alert the user.
+
+        if (!originalMessage && payload.action !== ComponentAction.Reply) {
```

Comment:
In which cases are we going to receive a message from the `ComponentManager` that is not a `ComponentAction.Reply`?

According to our test suite, the `ComponentRelay` can receive a message with the following actions:

- `ComponentAction.ComponentRegistered`: https://github.com/standardnotes/snjs/blob/9e5a65166c2e89524d17593d656cd198900cba1a/packages/snjs/lib/services/component_manager.ts#L1482
- `ComponentAction.ActivateThemes`: https://github.com/standardnotes/snjs/blob/9e5a65166c2e89524d17593d656cd198900cba1a/packages/snjs/lib/services/component_manager.ts#L466
- `ComponentAction.Reply`: https://github.com/standardnotes/snjs/blob/9e5a65166c2e89524d17593d656cd198900cba1a/packages/snjs/lib/services/component_manager.ts#L622

Reply:
You're saying that a reply is the most common case and that we don't need this condition to execute the code?

Reply:
Yeah, most of the messages have the `ComponentAction.Reply`, so the alert would never show.

Reply:
But I suppose that if the message is ComponentRegistered and we can't find the original message, it may be a bad sign and worth alerting the user of.

Reply:
Mmm... `ComponentRegistered` message is sent from `snjs` through the `registerComponentWindow` function and not from the component relay, so there shouldn't be any original message in `this.sentMessages`.

As I see it, the conditional `!originalMessage && payload.action !== ComponentAction.Reply` will always return `false` because all messages should be of `ComponentAction.Reply` inside the `default` case.

Reply:
Ok in that case we can keep the existing functionality that simply logs this message instead of presenting an alert.